src/components/verification-grid/css/style.css

```diff
@@ -35,8 +35,15 @@
   user-select: none;
   justify-content: center;
   align-items: stretch;
-  margin-top: var(--oe-spacing);
-  margin-bottom: var(--oe-spacing);
+
+  /*
+    We use padding instead of margin here because the default background color
+    for fullscreen mode is black (turning margin black).
+    If margin was used, there would be black bars around the verification grid
+    appearing as if the verification grid was touching the edge of the screen
+  */
+  padding-top: var(--oe-spacing);
+  padding-bottom: var(--oe-spacing);
```

Comment:
sounds like the correct fix is to style a background color of some other element

Reply:
This is how I originally tried to fix this problem

Remember that we are creating styles for web components, so we can't change the `body` background color or any other element that we don't control

In my mind the `.verification-grid` is the "top most" level element that we control.

---

Additionally: The fact that the verification-grids background did not extend all the way to the edge of the screen (even in windowed mode) is a bug.

Imagine that the background color was something more distinct from the default background color (white)

You'd notice right away that the background color didn't cover the entire screen

Reply:
>Remember that we are creating styles for web components, so we can't change the body background color or any other element that we don't control

> In my mind the .verification-grid is the "top most" level element that we control.

Valid, but then why are we "fixing" this at all?

You know what I think it's your comment that's misleading. It's pretty normal for an element to have some padding inside. Why talk about margins and background colors at all? That's the host's concern.

---

So, is the issue here that the actual element is being made full screen?  And because it is removed from it's host then we get a black background?

Seems like this should be the fix: https://stackoverflow.com/a/75765926/224512